apps/synapse-portal-framework/src/components/SectionLayout.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Container, ContainerProps } from '@mui/material'
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+import { HelpPopover } from 'synapse-react-client'
+import { scrollToWithOffset } from '../utils'
+
+type SectionLayoutProps = React.PropsWithChildren<{
+  ContainerProps?: ContainerProps
+  title?: string
+  centerTitle?: boolean
+  subtitle?: string
+  helpText?: string
+}>
+
+export function SectionLayout(props: SectionLayoutProps) {
```

Comment:
Extracted from `ComponentRenderer`, a layout component for non-explore and non-detailspage content